File: tmdb-core/src/main/java/dev/reuss/tmdb/domain/certifications/CertificationPaths.java
package dev.reuss.tmdb.domain.certifications;

final class CertificationPaths {

    private static final String CERTIFICATION = "/certification";

    private CertificationPaths() {
    }

    static String movieCertifications() {
        return CERTIFICATION + "/movie/list";
    }

    static String tvCertifications() {
        return CERTIFICATION + "/tv/list";
    }
}

File: tmdb-core/src/main/kotlin/dev/reuss/tmdb/domain/certifications/CertificationPaths.kt
package dev.reuss.tmdb.domain.certifications

import dev.reuss.tmdb.core.path.tmdbPath

internal object CertificationPaths {
    fun movieCertifications(): String = tmdbPath("certification", "movie", "list")

    fun tvCertifications(): String = tmdbPath("certification", "tv", "list")
}
